sql-common-clauses/sql-common-clauses_solution.sql

```diff
@@ -0,0 +1,42 @@
+#SELECT `SELECT` to bypass an in-built feature (use backticks)
+
+
+#1
+
+USE telecom;
+
+SELECT id, log_feature AS `log`, volume AS `vol`
+from log_feature;
+
+#Sorting
+#2
+SELECT id, resource_type
+from resource_type
+ORDER BY 1 ASC
+LIMIT 5;
+
+#3
+SELECT id, resource_type
+from resource_type
+ORDER BY 1, 2 ASC
+LIMIT 5;
+
+#4
+SELECT id, resource_type

```

Comment:
I think you swapped this one and the previous one. This is correct for the third problem, but the previous one isn't quite correct for the fourth problem.